rest/route/user.go

```diff
@@ -210,6 +210,29 @@ func (h *userPermissionsPostHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewTextInternalErrorResponse(err.Error())
 	}
 
+	// This is unfortunately very special-casey, but if the user has been granted permission to view/edit
+	// project settings, they also need to be given view access for the repo project, if applicable.
```

Comment:
Given this change, I think we should also update the text in the UI where admins are added 

> Admins for this branch will be able to edit branch settings. All admins will have access to create new projects on Evergreen **and view the repo if the project is attached**.

Reply:
we actually already have "and view repo settings" for attached projects:
![image](https://github.com/user-attachments/assets/faf797f3-2f96-4de8-a255-a4ee1b6f1124)